x/vesting/keeper/integration_test.go

```diff
@@ -514,11 +843,16 @@ var _ = Describe("Clawback Vesting Accounts - claw back tokens", Ordered, func()
 	})
 })
 
+func createAddressKey() (sdk.AccAddress, *ethsecp256k1.PrivKey) {
+	address, privKey := tests.NewAddrKey()
+	return address.Bytes(), &ethsecp256k1.PrivKey{Key: privKey.Bytes()}
+}
+
```

Comment:
please undo. This util func shouldn't be here. If you need to access the ethsecp key then we should update the `tests.NewAddressKey` to return that instead of the interface